packages/postgres-database/requirements/_base.in

```diff
@@ -6,5 +6,5 @@
 
 
 alembic
-sqlalchemy[postgresql_psycopg2binary,asyncio] # SEE extras in https://github.com/sqlalchemy/sqlalchemy/blob/main/setup.cfg#L43
+sqlalchemy[postgresql_psycopg2binary,asyncio,mypy] # SEE extras in https://github.com/sqlalchemy/sqlalchemy/blob/main/setup.cfg#L43
```

Comment:
Wy not adding mypy extra `_tools.in` or `_tests.in`

```requirements.txt
sqlalchemy[mypy]
```

Reply:
because it was simpler like this, but I can move it